### DISCUSSION TITLE
Loading both WES and WGS data for a family

### DISCUSSION THREAD

**Discussion by anjalijain22** [broadinstitute/seqr#3571] [Q&A] [answered]
Hi Hana,

I have a family that has both WES and WGS data. I was able to load both the datasets for the family but I was wondering -
1. If there is a way to conduct a separate WES and WGS analysis of the family? From what I understand so far when I do a family variant search, seqr searches both the WES and WGS datasets and if a variant is called in both datasets it is denoted with a "+" icon to reveal more information. But is there a way for me to only search the WGS dataset and not the WES dataset (or vice versa) while having both loaded on seqr?
2. How can I add both the WES and WGS BAM files for viewing in seqr? I tried uploading a file with the IGV paths in the below format but it only loaded the cram file:
<html>

<head>

</head>

<body>

sample_123 | /readviz/sample123_WES.bam
-- | --
sample_123 | /readviz/sample123_WGS.cram

</body>

</html>

Many thanks,
Anjali

**Comment by hanars** [comment, marked as answer]
Loading both WES and WGS data to the same family is not well supported behavior in seqr. Internally, we make separate projects for WES and WGS data and load them separately to avoid these issues. There is no way to do any of what you are asking about in this ticket at this time if you have the data in the same project

**Reply by anjalijain22** [reply]
I see! In that case, I'll create a separate project for WGS. 

Thanks for your reply, appreciate it!
